packages/firestore/src/util/log.ts

```diff
@@ -28,6 +28,20 @@ export function getLogLevel(): LogLevel {
   return logClient.logLevel;
 }
 
+/**
+ * Sets the verbosity of Cloud Firestore logs (debug, error, or silent).
+ *
+ * @param logLevel
+ *   The verbosity you set for activity and error logging. Can be any of
+ *   the following values:
+ *
+ *   <ul>
+ *     <li><code>debug</code> for the most verbose logging level, primarily for
```

Comment:
Should these `<code>` spans just be backticks?

Reply:
Fixed